daemon/cmd/daemon_main.go

```diff
@@ -1053,6 +1053,10 @@ func initializeFlags() {
 	flags.Bool(option.EnableBGPControlPlane, false, "Enable the BGP control plane.")
 	option.BindEnv(Vp, option.EnableBGPControlPlane)
 
+	flags.Bool(option.EnableStaleCiliumEndpointSliceCleanup, true, "Enables running cleanup init procedure of local CiliumEndpoints which are not being managed.")
+	flags.MarkHidden(option.EnableStaleCiliumEndpointSliceCleanup)
+	option.BindEnv(Vp, option.EnableStaleCiliumEndpointSliceCleanup)
+
```

Comment:
Shouldn't the flag simply be called `EnableStaleCiliumEndpointCleanup`? Why the reference to "slice"? It makes it sound like it's specific to CES and not CEPs.

Reply:
weird, I thought I changed that, I got myself into a weird git rebase mess last night and must have lost the change somehow. Ill fix it again.